packages/big-design/src/components/Radio/styled.tsx

```diff
@@ -9,11 +9,25 @@ interface StyledRadioProps {
   disabled?: boolean;
 }
 
+export const RadioLabelContainer = styled.div`
+  margin-left: ${({ theme }) => theme.spacing.xSmall};
+`;
+
+RadioLabelContainer.defaultProps = { theme: defaultTheme };
+
 export const RadioContainer = styled.div`
-  align-items: center;
+  align-items: flex-start;
   display: flex;
 `;
 
+export const RadioDescription = styled.span`
```

Comment:
Consider using the `StyleableSmall` in `Typography/private`. See how we consume it in the Messaging components (`Alert`, `InlineAlert`, and `Message`).